Dockerfile.local

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG=golang:1.22.10-alpine3.19
+ARG GOLANG=golang:1.22.12-alpine3.19
```

Comment:
Any reason not to also use `-alpine3.20` here?

Reply:
updated